services/slack.js

```diff
@@ -5,18 +5,23 @@ var robot = new SlackRobot(env.SLACK_TOKEN);
 var logFactory = require('../helpers/log.js');
 var log = logFactory.create("slack/slack-robot");
 
-var achievement = require('../models/achievement.js');
+var googleCalendar = require('../services/calendar.js');
 
-// will post 'world' text as bot when receiving 'hello' message
-// in channel, group, or direct message
+  // will post 'world' text as bot when receiving 'hello' message
+  // in channel, group, or direct message
 robot.listen(">:message(.+)", function (req, res) {
 
     var user = req.from.name;
     var message = req.params.message;
     log.info("Got message from slack: "+ message);
 
-    achievement.save(user,message);
-    res.text("Achievement recorded").send();
+    googleCalendar.recordEvent(message, function(err, data){
```

Comment:
Append slack name from the user to the message. Also consider another format, to post achievements for others:

`> @marbu won the lottery`